### ISSUE TITLE
Bug in ManualCube rendering - buffer is destroyed before used

### ISSUE THREAD

**Issue by RonenNess** [LaudableBauble/Insipidus#3]
In file: https://github.com/LaudableBauble/Insipidus/blob/master/InsipidusEngine/InsipidusEngine/ManualCube.cs

There are the following lines inside RenderToDevice:

```

            // Create the shape buffer and dispose of it to prevent out of memory
            using (VertexBuffer buffer = new VertexBuffer(
                device,
                VertexPositionNormalTexture.VertexDeclaration,
                NUM_VERTICES,
                BufferUsage.WriteOnly))
            {
                // Load the buffer
                buffer.SetData(_vertices);

                // Send the vertex buffer to the device
                device.SetVertexBuffer(buffer);
            }

            // Draw the primitives from the vertex buffer to the device as triangles
            device.DrawPrimitives(PrimitiveType.TriangleList, 0, NUM_TRIANGLES);
```

When running this with MonoGame >3.5 it cause exception, because the final line

`device.DrawPrimitives(PrimitiveType.TriangleList, 0, NUM_TRIANGLES);`

Is outside the `using` block, so by the time its called, the temp VertexBuffer 'buffer' is already destroyed and there's corruption data.

tl;dr, the line 

`device.DrawPrimitives(PrimitiveType.TriangleList, 0, NUM_TRIANGLES);`

Should be inside the
`using (VertexBuffer buffer = ....`

Block.
